src/applications/vaos/components/calendar/CalendarWidget.jsx

```diff
@@ -335,6 +335,7 @@ export default class CalendarWidget extends Component {
                 <div
```

Comment:
Let's add an `aria-describedby="h2-unique-id"` here an an ID to the H2 on line 307 so we can hook into the heading as a table "caption".